debian/changelog

```diff
@@ -1,3 +1,15 @@
+rdma-core (21.0-2) UNRELEASED; urgency=medium
+
+  * drop ibacm sysV init script to avoid issues with the sysV to
+    systemd wrapper starting the service instead of the socket (LP: #1794825)
+    - d/rules: drop dh_installinit call for ibacm
+    - d/rules: let dh_installsystemd not start ibacm.service
+    - d/ibacm.maintscript: remove old conffile
+    - d/ibacm.install: no more install sysV script
+    - d/not-installed: mark etc/init.d/ibacm as deliberately not installed
+
```

Comment:
We are not maintaining the changelog here, it is kept in the debian branch for debain. This just has one entry with the right version to make it build right. So drop this change